src/main/resources/db/migration/V4_3__alter_motedeltaker_arbeidstaker_varsel_add_column_journalpost_id.sql

```diff
@@ -0,0 +1,9 @@
+-- ROLLBACK-START
+------------------
+-- ALTER TABLE MOTEDELTAKER_ARBEIDSTAKER_VARSEL DROP MOTEDELTAKER_ARBEIDSTAKER_VARSEL;
```

Comment:
Blir rollback her riktig? Blir ikke det noe ala:

ALTER TABLE MOTEDELTAKER_ARBEIDSTAKER_VARSEL DROP COLUMN IF EXISTS journalpost_id;

Reply:
Kan det oppstå rollback her uten at kolonnen har blitt opprettet? Ettersom det kun er 1 sql-linje.

Reply:
Det vet jeg ikke (kanskje ikke?), bare lurte litt på om det var riktig med "DROP MOTEDELTAKER_ARBEIDSTAKER_VARSEL" vs "DROP COLUMN journalpost_id"

Reply:
Endret rollback-linjen nå.

Reply:
Kjørte en redigering på svaret mitt, for jeg så ikke først hvor feil linjen var..

Reply:
Skjønner :D

Reply:
Approved